synthtool/gcp/templates/node_library/.github/workflows/ci.yaml

```diff
@@ -9,10 +9,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14, 15]
+        node: [10, 12, 14, 16, 17]
```

Comment:
Since 17 is only stable for 6 months, I'm tempted to just add `16`? I'm cautious of being somewhat sparing with our test matrix size, in case we do eventually start bumping into any quotas on GitHub's side.

Reply:
Entirely your call.  Traditionally I've liked having "latest", whatever that may be, in the matrix.  Invariably, someone runs with the hottest bits here, and if something breaks or causes trouble, we probably need to know about it.  That having been said, such occurrences are rare.  I'm cool either way, but for now I'll just remove.